itou/approvals/models.py

```diff
@@ -537,6 +537,17 @@ def clean(self):
                         )
                     }
                 )
+            next_min_start_at = self.next_min_start_at(self.approval)
+            if self.start_at < next_min_start_at:
+                raise ValidationError(
+                    {
+                        "start_at": (
+                            f"Pour la date de début de suspension, vous pouvez remonter "
+                            f"{self.MAX_RETROACTIVITY_DURATION_DAYS} jours avant la date du jour."
+                            f"Date de début minimum: {next_min_start_at.strftime('%d/%m/%Y')}."
```

Comment:
```suggestion
                            f"Date de début minimum : {next_min_start_at.strftime('%d/%m/%Y')}."
```

Petite subtilité de la langue française : [il y a une espace avant et après les deux points](https://www.btb.termiumplus.gc.ca/redac-chap?lang=fra&lettr=chapsect6&info0=6.13) (à l'inverse de l'anglais où il n'y a pas d'espace avant).